lib/game.js

```diff
@@ -142,12 +143,24 @@ function clearScoreAndRound() {
   displayScore();
 }
 
+function displayInstructions() {
+  $('#instructions').html("<h4>" + "Instructions" + "</h4>" + 
+                          "<p>" + "Left and right arrow keys to move the \"Jenny\"" + "</p>" +
+                          "<p>" + "Spacebar to drop nets on shrimp & score points" + "</p>" +
+                          "<p>" + "Pass Round 1 with 1500 points" + "</p>" + 
+                          "<p>" + "Pass Round 2 with 2500 points" + "</p>" +
+                          "<p>" + "Win the game with 3500 points!" + "</p>" +
+                          "<p>" + "Hint: bottom shrimp are worth more points" + "</p>" +
+                          "<h5>" + "Click the water & press enter" + "</h5>" +
```

Comment:
Clicking the water and also pressing ENTER was pretty confusing for me when I pulled this down this morning to test it out. If it's possible to simplify starting the game for the user that would be :100: (time permitting)